jaeger-zipkin/src/test/java/com/uber/jaeger/senders/zipkin/ZipkinSenderTest.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2016, Uber Technologies, Inc
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.uber.jaeger.senders.zipkin;
+
+import com.uber.jaeger.Span;
+import com.uber.jaeger.SpanContext;
+import com.uber.jaeger.Tracer;
+import com.uber.jaeger.exceptions.SenderException;
+import com.uber.jaeger.metrics.InMemoryStatsReporter;
+import com.uber.jaeger.reporters.InMemoryReporter;
+import com.uber.jaeger.reporters.Reporter;
+import com.uber.jaeger.reporters.protocols.ThriftSpanConverter;
+import com.uber.jaeger.samplers.ConstSampler;
+import org.apache.thrift.protocol.TBinaryProtocol;
+import org.apache.thrift.transport.AutoExpandingBufferWriteTransport;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import zipkin.junit.ZipkinRule;
+import zipkin.reporter.MessageEncoder;
+import zipkin.reporter.urlconnection.URLConnectionSender;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class ZipkinSenderTest {
```

Comment:
made a general note.. I think there's a couple ways to do this.. maybe I can do it without affecting any of the main uber interfaces (ie by making an abstract method on the test case). I'll own at least trying this as a follow-up!